contrib/lib/tests/databases.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(use_extern_macros)]
+
+#[macro_use] extern crate rocket;
```

Comment:
I don't think we need this `macro_use` here, do we?

Reply:
Nope.